app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -283,7 +283,7 @@ class AdministratorControlsActivityTest {
   }
 
   @Test
-  fun testAdministratorControlsFragment_loadFragment_topicUpdateOnWifiSwitchIsNotChecked_autoUpdateTopicSwitchIsNotChecked() { // ktlint-disable max-line-length
+  fun testAdminControlsFragment_OnWifiSwitchIsNotChecked_autoUpdateTopicSwitchIsNotChecked() {
```

Comment:
Always make sure that the first character after `_` is lower case. 

Suggestion: 
`testAdminCtrl_topicUpdateOnWifiSwitchIsNotChecked_autoUpdateTopicSwitchIsNotChecked `

Reply:
In this function @rt4914, shouldn't we have AdminCtrlFragment instead of omitting "fragment"?

Reply:
In this case/file, its file to omit the `fragment` word.